tests/test_jenkins.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+
+@pytest.fixture()
+def AnsibleDefaults(Ansible):
+
+    """ Load default variables into dictionary.
+
+    Args:
+        Ansible - Requires the ansible connection backend.
+    """
+    return Ansible("include_vars", "./defaults/main.yml")["ansible_facts"]
+
+
+@pytest.fixture()
+def AnsibleFacts(Ansible):
+
+    """ Load ansible facts into a dictionary.
+
+    Args:
+        Ansible - Requires ansible connection backend.
+    """
+    return Ansible("setup")["ansible_facts"]
+
+
+def test_jenkins_package(Package, AnsibleDefaults):
+
+    """ Ensure jenkins package is both installed and at the proper major
+    version.
+
+    Args:
+        Package - module to determine package install status and version.
+        AnsibleDefaults - module to pull the target version of Jenkins.
+
+    Raises:
+        AssertionError if package is not installed or the wrong version.
+    """
+    jenkins = Package("jenkins")
+    assert jenkins.is_installed
+    jenkins_major_version = AnsibleDefaults["jenkins_major_version"]
+    assert jenkins.version.startswith(jenkins_major_version)
```

Comment:
Can we do something against `jenkins_version` here that takes version information before the `*`?

```python
assert jenkins.version.startswith(jenkins_version.partition('*')[0])
```

**Example**

```python
In [1]: jenkins_version = "2.*"

In [2]: jenkins_version.partition("*")[0]
Out[2]: '2.'
```